src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
 // write your code here
+// Отримуємо всі елементи <span> з класом "population"
+const populationSpans = document.querySelectorAll('span.population');
+// Масив для збереження числових значень
+const populationValues = [];
+
+// Проходимо через кожен елемент, перевіряємо та конвертуємо текст у число
+populationSpans.forEach((span) => {
+  const populationText = span.textContent.replace(/,/g, '');
+  const populationNumber = parseFloat(populationText);
```

Comment:
You are using parseFloat to convert a string to a number. This is generally fine, but keep in mind that parseFloat will convert a string to a floating point number, which may not be what you want if you are dealing with population numbers. Consider using parseInt instead if you want to deal with whole numbers.